packages/framework/fluid-static/src/fluidContainer.ts

```diff
@@ -10,18 +10,50 @@ import { AttachState } from "@fluidframework/container-definitions";
 import { LoadableObjectClass, LoadableObjectRecord } from "./types";
 import { RootDataObject } from "./rootDataObject";
 
+/**
+ * Events emitted from IFluidContainer.
+ */
 export interface IFluidContainerEvents extends IEvent {
     (event: "connected" | "dispose" | "disconnected", listener: () => void): void;
 }
 
+/**
+ * The IFluidContainer provides an entrypoint into the client side of collaborative Fluid data.  It provides access
+ * to the data as well as status on the collaboration session.
+ */
 export interface IFluidContainer extends IEventProvider<IFluidContainerEvents> {
+    /**
```

Comment:
Looks like `attach()` wasn't added to the interface. Can you add it here.

Reply:
Got it prepped in another branch, was gonna put it up immediately following this change.